src/index.js

```diff
@@ -1,13 +1,120 @@
-export default function BaseballGame() {
-  this.play = function (computerInputNumbers, userInputNumbers) {
-    return "결과 값 String";
-  };
+let comNumbers; // 컴퓨터의 3자리 숫자를 담을 변수 선언
+
+function setRandomNumber(num) {
+  // 3자리 랜덤 숫자를 문자열로 리턴하는 함수
+  num = "000"; // 기본 문자열 '000'설정
+  while (
+    // num의 100, 10, 1의 자리가 전부 다른 숫자가 되도록 작동하는 반복문
+    num[0] === num[1] ||
+    num[0] === num[2] ||
+    num[1] === num[2]
+  ) {
+    let randomNumber = String(Math.floor(Math.random() * 1000)); // 3자리 숫자 랜덤숫자 문자열 생성
+    if (randomNumber < 100 || randomNumber.indexOf("0") !== -1) {
+      // 수가 100보다 작거나 0이 포함될 시 반복문 재시작
+      continue;
+    }
+    return randomNumber; // 모든 조건에 맞으면 해당 숫자 문자열을 반환
+  }
+}
+
+comNumbers = setRandomNumber(comNumbers); // 3자리 숫자 설정 함수 호출 후 comNumber에 대입
+
+///////////////////////////////////////////////////////////////////////////////////////////
+
+function newGame(e) {
+  // 새로운 게임을 시작 함수
+  resultContainer.innerHTML = ""; // 결과값 초기화
+  comNumbers = setRandomNumber(comNumbers); // 컴퓨터 랜덤 변수 재설정
+}
+
+function submitAnswer(e) {
+  // 버튼 대신 endter를 눌렀을 때
+  if (e.which === 13) {
+    // enter 또는 numpad 버튼을 눌렀을 때
+    e.preventDefault(); // form 태그 안에 있는 버튼의 제출을 막는다.
+    playGame(); // '확인' 버튼을 눌렀을 때 발생하는 이벤트를 실행한다.
+  }
+}
+
+function checkInput(input) {
+  // 사용자 입력값이 중복이 되지않는 3자리 정수 인지를 확인
+  if (
+    Boolean(+input) === false ||
```

Comment:
그리고 보통은 이런식으로 축약을 많이해서 표현합니다
```suggestion
    !Boolean(+input) ||
```

Reply:
Javascript의 falsy, truthy 에 대해서 한번 찾아보셔도 좋을거 같습니다